tests/functional/wrappers/test_ytoken.py

```diff
@@ -9,28 +9,16 @@ def test_config(gov, token, vault, registry, ytoken):
     # No vault added to the registry yet, so these methods should fail
     assert registry.numVaults(token) == 0
 
-    with brownie.reverts():
-        assert ytoken.name()
-
-    with brownie.reverts():
-        assert ytoken.symbol()
-
-    with brownie.reverts():
-        assert ytoken.decimals()
-
-    with brownie.reverts():
-        ytoken.bestVault()
-
     # This won't revert though, there's no Vaults yet
     assert ytoken.allVaults() == []
 
     # Now they work when we have a Vault
     registry.newRelease(vault, {"from": gov})
     registry.endorseVault(vault, {"from": gov})
     assert ytoken.bestVault() == vault
-    assert ytoken.name() == vault.name()
-    assert ytoken.symbol() == vault.symbol()
-    assert ytoken.decimals() == vault.decimals()
+    assert ytoken.name() == "Yearn " + token.name()
+    assert ytoken.symbol() == "y" + token.symbol()
+    assert ytoken.decimals() == token.decimals()
```

Comment:
Put these whwre the above are. In the test flow, things that are set by default are in the first batch of asserts